src/components/Login/Login.js

```diff
@@ -0,0 +1,110 @@
+import React, { useState } from 'react'
```

Comment:
Not sure we need this file. Have you pulled from develop recently?

Reply:
I've pulled from develop already. Why dont we need the login file?